### DISCUSSION TITLE
how to remove Toolbar Windows Explorer

### DISCUSSION THREAD

**Discussion by ObiKusakabe** [Open-Shell/Open-Shell-Menu#1007] [Q&A]
how to remove Toolbar Windows Explorer, idk how to remove it, i've also deleted the open shell, but it didn't work

**Comment by Raylopi** [comment]
Right click on it and deselect it

**Comment by adamlove86** [comment]
I've tried this but it keeps coming back the next time an explorer window opens.

**Reply by ge0rdi** [reply]
You can run `Open-Shell` installer, select `Change` and then remove `Classic Explorer` feature.

**Comment by Jamesstevens-3rd** [comment]
if you want to keep the start menu but deactivate Classic Explorer this should work but it will remove all the Explorer features.

look for **Internet Options** in the **Control Panel** or type **Internet Options** in the **start menu search box**, then go to the **Programs** tab and **Manage add-ons**  disable the following:

Classic Explorer Bar
ExplorerBHO Class

Disabling one should disable the other, then restart you computer and check if they are both still disabled.
